src/main/java/com/google/api/codegen/gapic/GapicGeneratorFactory.java

```diff
@@ -373,7 +374,19 @@
                 .build();
         generators.add(testGenerator);
       }
-
+      if (artifactFlags.sampleGeneratorEnabled()) {
```

Comment:
I looked at google-cloud-lang repos and it seems none of them put samples in the same repo... All of them put samples in a different repo called lang-docs-samples. Can you clarify more about what "living elsewhere" means?

Reply:
The plan is that for most languages (unless there is a strong language convention not to) the standalone samples will live in a separate, top-level directory `samples/` so that they are browseable with the client libraries. We have not started publishing samples yet, which is why you don't see them there yet....

Reply:
Ok. I will change the behavior to default first and check with yoshi team to see if they are OK with the default behavior.

Reply:
We can just have the samples generated with the client library for now (development) and then as they get ready to publish we can tweak if they need it changed.